ddbToEsDlqLambda/loggerBuilder.ts

```diff
@@ -0,0 +1,9 @@
+import { makeLogger } from 'fhir-works-on-aws-interface';
+
+const componentLogger = makeLogger({
+    component: 'persistence',
```

Comment:
We should pass through or set the log level so we can increase/decrease the noise as needed in prod.

https://github.com/awslabs/fhir-works-on-aws-interface/blob/mainline/src/loggerBuilder.ts#L40

Reply:
I can add LOG_LEVEL environment variable